### DISCUSSION TITLE
can I override filament base Layout

### DISCUSSION THREAD

**Discussion by ayemin-tun** [filamentphp/filament#9330] [General]
### Description

can i override this baselayout in filament like override logo .
![Screenshot (151)](https://github.com/filamentphp/filament/assets/76741291/78a3bac9-be82-4221-ac8b-cc0bd5b5fcb0)

**Comment by VincvanAs** [comment]
You should create a custom theme

https://filamentphp.com/docs/3.x/panels/themes#adding-a-logo
